api/app/signals/apps/questionnaires/templates/questionnaires/swagger/public_openapi.yaml

```diff
@@ -600,6 +591,164 @@ components:
           pattern: '^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$'
           example: '00000000-0000-0000-0000-000000000000'
 
+    AnswerBoolean:
+      type: boolean
+      example: true
+
+    AnswerDate:
+      type: string
+      format: date
+      example: 2022-09-13
+
+    AnswerTime:
+      type: string
+      format: time
+      example: 00:00:00
+
+    AnswerDateTime:
+      type: string
+      format: date-time
+      example: 2022-09-13T00:00:00
+
+    AnswerFloat:
+      type: number
+      format: float
+      example: 3.14159
+
+    AnswerInteger:
+      type: integer
+      example: -1234
+
+    AnswerPositiveInteger:
+      type: integer
+      minimum: 0
+      example: 5678
+
+    AnswerPlainText:
+      type: string
+      example: Just an example text that can be submitted.
+
+    AnswerEmail:
+      type: string
+      format: email
+      example:
+
+    AnswerDutchTelephoneNumber:
+      type: string
+      pattern: ^(?:0|(?:\+|00) ?31 ?)(?:(?:[1-9] ?(?:[0-9] ?){8})|(?:6 ?-? ?[1-9] ?(?:[0-9] ?){7})|(?:[1,2,3,4,5,7,8,9]\d ?-? ?[1-9] ?(?:[0-9] ?){6})|(?:[1,2,3,4,5,7,8,9]\d{2} ?-? ?[1-9] ?(?:[0-9] ?){5}))$
+      example: 06 12 34 56 78
+
+    AnswerSelectedObject:
+      type: object
+      properties:
+        id:
+          oneOf:
+            - type: integer
+            - type: string
+          nullable: true
+          example: 1234
+        type:
+          type: string
+          example: container
+        onMap:
+          type: boolean
+          example: false
+        coordinates:
+          type: object
```

Comment:
Add a description explaining which coordinate system we use (WGS84 a.k.a. EPSG:4326).